maintainers/maintainer-list.nix

```diff
@@ -1280,6 +1280,12 @@
     githubId = 64804;
     name = "Dennis Gosnell";
   };
+  ccellado = {
```

Comment:
Could you make this a separate commit?

Reply:
the commit should be named `maintainers: add ccellado`, see
https://github.com/ccellado/nixpkgs/commits/glpaper-unstable/maintainers/maintainer-list.nix

Reply:
Changed